waspc/src/Wasp/Util.hs

```diff
@@ -118,3 +121,13 @@ insertAt :: [a] -> Int -> [a] -> [a]
 insertAt theInsert idx host =
   let (before, after) = splitAt idx host
    in before ++ theInsert ++ after
+
+infixr 5 <++>
```

Comment:
Cool -> how did you know which `infixr` priority to put?

Reply:
I just did `:t (++)` in ghci for some initial inspiration, plus checked out Hoogle for more. Most were 5-7.